packages/terra-search-field/src/terra-dev-site/doc/example/SearchFieldBasic.jsx

```diff
@@ -1,9 +1,38 @@
 import React from 'react';
 // eslint-disable-next-line import/no-extraneous-dependencies, import/no-unresolved, import/extensions
-import SearchFieldExampleTemplate from 'terra-search-field/lib/terra-dev-site/doc/example/SearchFieldExampleTemplate';
+import SearchField from 'terra-search-field/lib/SearchField';
 
-const SearchFieldExample = () => (
-  <SearchFieldExampleTemplate />
-);
+class SearchFieldBasic extends React.Component {
```

Comment:
This example is simple enough I think it could be written with hooks: https://reactjs.org/docs/hooks-intro.html

Reply:
Here's an example of how it could be switched to use hooks:

```jsx
import React, { useState } from 'react';
// eslint-disable-next-line import/no-extraneous-dependencies, import/no-unresolved, import/extensions
import SearchField from 'terra-search-field/lib/SearchField';

const INVALID_MESSAGE = 'The default minimum search length is 2.';

const SearchFieldExample = () => {
  const [searchText, setSearchText] = useState('');

  const message = searchText.length > 2 ? `Search text: ${searchText}` : INVALID_MESSAGE;

  return (
    <div>
      <p>{message}</p>
      <SearchField onSearch={setSearchText} onInvalidSearch={setSearchText} />
    </div>
  );
};

export default SearchFieldExample;
```